pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -97,6 +97,7 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, in io.Reader, out io.
 	cmd.Flags().StringSliceVar(&config.SourceRepositories, "code", config.SourceRepositories, "Source code in the build configuration.")
 	cmd.Flags().StringSliceVarP(&config.ImageStreams, "image", "i", config.ImageStreams, "Name of an image stream to to use as a builder.")
 	cmd.Flags().StringSliceVar(&config.DockerImages, "docker-image", config.DockerImages, "Name of a Docker image to use as a builder.")
+	cmd.Flags().StringSliceVar(&config.Secrets, "secret", config.Secrets, "Secret and destination to use as an input for the build.")
```

Comment:
@csrwng the API field is called "secrets" ;-) But we can use `--source-secret` here to match with the struct we use. @fabianofranz @smarterclayton thoughts?